src/tooltipBuilder.ts

```diff
@@ -49,7 +49,6 @@ export function createTooltipInfo(
     localizationManager: ILocalizationManager,
     value?: any,
     seriesIndex?: number,
-    categoryIndex?: number,
```

Comment:
why did you remove categoryIndex here?

Reply:
This argument isn't used in function body